### ISSUE TITLE
Console Warning with the customization attribute, batteryMeter with the noOfCells is >1 is creating warnings

### ISSUE THREAD

**Issue by simmy-reach88** [Umerbhat/react-battery-gauge#9]
This is similar to this issue but associated with a different property:  https://github.com/Umerbhat/react-battery-gauge/issues/6

I am using the BatteryGauge component with batteryMeter attribute    
```
const batterySegmentCount = 5 
batteryMeter: {
            fill: 'green',
            noOfCells: batterySegmentCount,
            lowBatteryValue: batteryIncrement + 1,
 },
```
noOfCells is having a value of 5. For any value > 1, it is throwing the following warnings:

```react.development.js:209 Warning: Each child in a list should have a unique "key" prop.

Check the render method of `BatteryLevel`. See https://reactjs.org/link/warning-keys for more information.
    at rect
    at BatteryLevel (http://localhost:4200/vendors-node_modules_react-battery-gauge_dist_react-battery-gauge_esm_js.js:277:26)
    at g
    at svg
    at CanvasProvider (http://localhost:4200/vendors-node_modules_react-battery-gauge_dist_react-battery-gauge_esm_js.js:198:23)
    at Canvas (http://localhost:4200/vendors-node_modules_react-battery-gauge_dist_react-battery-gauge_esm_js.js:207:24)
    at BatteryGauge (http://localhost:4200/vendors-node_modules_react-battery-gauge_dist_react-battery-gauge_esm_js.js:576:24)
```

Another warning that is showing in console

```
react-dom.development.js:798 Error: <rect> attribute width: A negative value is not valid. ("-0.5")
```
On inspect I don't see any rect element with -ve value but when I am using the noOfCells>1 ,warning appears.

**Comment by Umerbhat** [comment]
We appreciate your contribution. We will investigate the warning messages, and we'll work on releasing a fix for it soon. Thank you for your patience!

**Comment by Umerbhat** [comment]
Fixed in `v1.0.6`
